src/components/ProjectListItem.tsx

```diff
@@ -37,7 +37,7 @@ const ProjectListItem: React.FC<ProjectListItemProps> = ({
           margin: isMobileScreen ? '16px 0' : '16px',
           cursor: 'pointer',
           borderRadius: '15px',
-          minHeight: '500px',
+          height: 'calc(100% - 32px)'
```

Comment:
I think you can achieve this by applying flex to ProjectListItem `<Grid item xs={12} sm={6} md={4} sx={{ display: 'flex' }}>` instead of calc.

Reply:
Thanks @buberdds  - changed!